molgenis-data-rest/src/main/java/org/molgenis/data/rest/v2/AttributeResponseV2.java

```diff
@@ -43,12 +43,17 @@
 	private String nullableExpression;
 	private String visibleExpression;
 	private String validationExpression;
+	private List<CategoricalOptionV2> categoricalOptions;
 
 	/**
-	 * @param fetch set of lowercase attribute names to include in response
+	 * Constructs AttributeResponseV2 using params
+	 *
+	 * @param fetch             set of lowercase attribute names to include in response
+	 * @param includeCategories if set to true fetches options list for CATEGORICAL and CATEGORICAL_MREF types,
+	 *                          if set to false references to the entities are returned
```

Comment:
this sounds like the reference to entities is not returned when flag is set to true

Suggestion: `@param includeCategories if set to true includes options list for CATEGORICAL and CATEGORICAL_MREF types in the attribute metadata`

don't need to describe the false scenario imo